atm.rb

```diff
@@ -0,0 +1,146 @@
+require "pry"
+class Menu
+  # puts "Enter username, please."
+  # username=gets.chomp
+  # puts "Enter password, please."
+  # password=gets.chomp
+  @@debitCard= []
```

Comment:
Does it make sense to store information like `@@debitCard` in a `Menu` class? `Menu` seems like something that should handle the display of a menu. Perhaps you should create an additional `Atm` class (or something that is finance-focused).